dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230922AddPublishDateToContentletVersionInfo.java

```diff
@@ -30,8 +42,18 @@ public void executeUpgrade() {
                 Logger.info(this, "Adding the 'publish_date' column to the 'contentlet_version_info' table");
                 dotConnect.executeStatement(ADD_PUBLISH_DATE_COLUMN);
                 Logger.info(this, "Setting the 'publish_date' column to the current version timestamp for existing content");
-                dotConnect.executeStatement("UPDATE contentlet_version_info SET publish_date = version_ts WHERE live_inode IS NOT NULL");
-            } catch (SQLException e) {
+                dotConnect.executeStatement(UPDATE_PUBLISH_DATE_COLUMN);
+
+                // Reindex contentlets to update the publish_date field
```

Comment:
Let's remove this logic since we shouldn't use API in UT... What we do is tell customers they need to reindex.

Reply:
API calls removed from the upgrade task.